src/CMakeLists.txt

```diff
@@ -223,6 +223,7 @@ target_include_directories(valhalla
     ${VALHALLA_SOURCE_DIR}/third_party/rapidjson/include
     ${VALHALLA_SOURCE_DIR}/third_party/date/include
     ${VALHALLA_SOURCE_DIR}/third_party/cpp-statsd-client/include
+    ${VALHALLA_SOURCE_DIR}/third_party/just_gtfs/include
```

Comment:
this should be in the private section. i hope we arent actually including this from any of our headers

Reply:
or wait, its already in mjolnir below.. why did we need it here again? i know we needed previously in the tests by why at the library level?

Reply:
Removed from this CMakeList - must have been a redundant include 😅